### DISCUSSION TITLE
I am getting error while trying to access over a track file for video subtitle

### DISCUSSION THREAD

**Discussion by MeherabTheLord** [community/community#38931] [New to GitHub]
Hello world,
I am trying to embed subtitle to a video file but while doing this i am facing some problem btw i am using local host as host server. While i am trying to load the subtitle file in console display and error which appear like "Unsafe attempt to load URL file:///E:/html/multimedia/videoSubtitle.vtt from frame with URL file:///E:/html/multimedia/index.html. 'file:' URLs are treated as unique security origins.";**_HOW TO FIX ?_** 
VIDEO file location:/E:/html/multimedia
SUBTITLE file location:/E:/html/multimedia

**Comment by jfullstackdev** [comment]
maybe this will help 

https://stackoverflow.com/questions/23256830/crossorigin-errors-when-loading-vtt-file

**Reply by MeherabTheLord** [reply]
I am practicing htlml in local host i think here i donot need any other or different domain access; When i am using this to my coding it appears like i donot evern have the video and video poster access from the local server. BTW thank you for the reply i will be happy to get other suggestion from you.
